tests/models/speecht5/test_tokenization_speecht5.py

```diff
@@ -202,3 +202,8 @@ def test_tokenizer_integration(self):
             revision="c5ef64c71905caeccde0e4462ef3f9077224c524",
             sequences=sequences,
         )
+
+    def test_encode_decode(self):
+        tokenizer = SpeechT5Tokenizer.from_pretrained("microsoft/speecht5_tts")
+        ids = tokenizer.encode("a = b")
+        self.assertEqual(tokenizer.decode(ids), "a <unk> b</s>")
```

Comment:
What exactly is this testing here? Is `=` a special token?

Reply:
Will add a bit of details !